rs/protobuf/def/state/metadata/v1/metadata.proto

```diff
@@ -245,7 +245,8 @@ message SubnetMetrics {
   types.v1.NominalCycles consumed_cycles_by_deleted_canisters = 1;
   types.v1.NominalCycles consumed_cycles_http_outcalls = 2;
   types.v1.NominalCycles consumed_cycles_ecdsa_outcalls = 3;
-  optional uint64 ecdsa_signature_agreements = 4;
```

Comment:
Just to verify, in the currently rolled out version, we never rely on that value anywhere? So if we need to downgrade to that version, this field being set to 0 will not cause any problems?

Reply:
Yes, that's fine. The value of `ecdsa_signature_agreements` has already been migrated to the new field `threshold_signature_agreements` on mainnet. See [here](https://victoria.ch1-obs1.dfinity.network/select/0/vmui/#/?g0.expr=max+by+%28ic_subnet%29+%28replicated_state_ecdsa_signature_agreements_total%7Bic_subnet%3D%22pzp6e-ekpqk-3c5x7-2h6so-njoeq-mt45d-h3h6c-q3mxf-vpeq5-fk5o7-yae%22%7D%29&g0.range_input=4h39m38s322ms&g0.end_input=2024-07-18T16%3A23%3A01&g0.relative_time=none&g0.tenantID=0&g1.expr=max+by+%28ic_subnet%29+%28replicated_state_threshold_signature_agreements_total%7Bic_subnet%3D%22pzp6e-ekpqk-3c5x7-2h6so-njoeq-mt45d-h3h6c-q3mxf-vpeq5-fk5o7-yae%22%7D%29&g1.range_input=4h39m38s322ms&g1.end_input=2024-07-18T16%3A23%3A01&g1.relative_time=none&g1.tenantID=0) for example.